ceph_deploy/hosts/debian/install.py

```diff
@@ -30,11 +30,11 @@ def install(distro, version_kind, version, adjust_repos, **kw):
         distro.packager.add_repo_gpg_key(gpg.url(key, protocol=protocol))
 
         if version_kind == 'stable':
-            url = 'http://download.ceph.com/debian-{version}/'.format(
+            url = 'https://download.ceph.com/debian-{version}/'.format(
```

Comment:
`apt-transport-https` is present on wheezy.  Let's just force install it right where we install `ca-certificates`?  Sounds like the way to go.